odoo/odoo
# [WIP] l10n_ec: Improvements for Ec

- Add Customer accounts receivable for point-of-sale module
- Add new accounting account group for customer checks
- Add 3210 tax code in report 103 of Ecuador
- Add 0% Non-Object VAT tax for dividends
- Add withholding tax on undistributed earnings for dividends
- Add the 0% withholding tax
- Add the following accounting accounts
  * 1102050103, Customer accounts receivable for point of sale
  * 1102050201, Customer checks to be deposited
  * 1102050202, Protested checks *11020705, Employee Loans
  * 11020706, Advance for overdraft in Role
  * 11050108, VAT paid on credit card withholding taxes
  * 11050108, VAT paid on credit card withholdings at source
  * 11050205,Bank Withholdings to be Reconciled Transitory Account
  * 110505, Credit Card Withholding Tax Paid at Source
  * 110506,Taxable Base of Withholding Taxes on Sales
  * 21140104,Taxable basis of withholdings on purchases
  * 5202040503,IESS contribution of personnel assumed by the company
  * 52011805, Credit card commissions
- Add the word "deprecated" to the name of the customer document types: Passport and UNKNOWN.
- Change Ecuador country configuration, to avoid the zip code to be mandatory when making Ecommerce transactions. This helps us to make ecommerce sales smoother. In addition to this we modify the way the name is configured in the report.
- Set default accounting taxes, for 15 % VAT
- add extra validation when getting the ats code from the contact, in case it is an End Consumer.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:




---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr
